firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionImpl.java

```diff
@@ -244,9 +244,8 @@ public Task<Void> signInTester() {
 
   @Override
   public void signOutTester() {
-    cachedNewRelease
-        .set(null)
-        .addOnSuccessListener(lightweightExecutor, unused -> signInStorage.setSignInStatus(false));
+    cachedNewRelease.set(null);
+    signInStorage.setSignInStatus(false);
```

Comment:
I also realized there is no point in waiting for one of these changes to be done before executing the other. They should both happen ASAP.